freeipa-client/build.gradle

```diff
@@ -26,6 +26,8 @@ dependencies {
   compile            group: 'org.glassfish.jersey.core',        name: 'jersey-common',               version: jerseyCoreVersion
   testImplementation group: 'org.mockito',                      name: 'mockito-core',                version: mockitoVersion
   testImplementation group: 'org.hamcrest',                     name: 'hamcrest',                    version: hamcrestVersion
+  compile            group: 'org.springframework.boot',         name: 'spring-boot-starter-web',     version: springBootVersion
```

Comment:
why do we need this dependency in the client?

Reply:
`freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java` is the first file in this module that depends on cluster proxy module and the cluster proxy module uses spring `@Component` annotations.